### ISSUE TITLE
[Feature request] Treating run duration as a summary metric

### ISSUE THREAD

**Issue by maria-korosteleva** [wandb/client#1368]
I'd like to be able to treat run duration like any other final metric, produces by the run (e.g. like a loss that I record)

Use-case example: 

I'm currently running a sweep (on a single machine) over my hyperparameters. Looking at the parameter importance table w.r.t. losses makes me feel that some parameters that influence the runtime significantly are not actually of much importance to loss values. If true, this might mean that I can make the model more efficient without losing much of accuracy. In this case, I'd need to see an importance plot for the run duration to quickly evaluate this suspicion (and hopefully save running time) =)

P.S. I understand that I can probably make a workaround by recording duration by myself from the training script, but why would I want to do that, if the info is already attached to the run?

**Comment by cvphelps** [comment]
Hi Maria, thanks for sharing this feature request. To clarify, the specific feature you'd like is to be able to select the run duration as a target metric in the parameter importance panel. Is that right?

**Comment by maria-korosteleva** [comment]
Hi @cvphelps!
No, what I wanted is the other way around:

1. Select duration as a target metric
2. See how my hyperparameters have an effect on this metric (e.g. larger number of Conv layers -> longer training?) 

Something like this. But your understanding also sounds interesting, actually 🤔

**Comment by sydholl** [comment]
In the past year we've majorly reworked the CLI and UI for Weights & Biases. We're closing issues older than 6 months. Please comment to reopen.

**Comment by KuSi833** [comment]
I would also be interested in something like this as @maria-korosteleva mentioned;
> 1. Select duration as a target metric
> 2. See how my hyperparameters have an effect on this metric (e.g. larger number of Conv layers -> longer training?)

**Comment by LucHayward** [comment]
I would also find it useful to be able to query the run duration from the api

**Comment by MaxSchambach** [comment]
Is this planned? It would be very usefule, plotting eval metrics over duration for examples can give valuable insights when comparing multiple runs or models against each other. It's a bit frustrating that this is currently not possible, despite the value being logged with the run (but not under `summary`).
